### PR TITLE
  installed tailwind and updated styles

### PR DESCRIPTION
**Pull request by jonathans199**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "./app/**/*.{js,ts,jsx,tsx,mdx}",
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
+    "./components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/**/*.{js,ts,jsx,tsx,mdx}",
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
```
Comment (zarela) [inline tailwind.config.js:13]: Missing end of line

**Review by zarela — CHANGES_REQUESTED** [review]
Requested a tiny change, just an end of line, but also since I merged prettier, you have to update your branch and solve the merge conflicts. Thank you! 😃

**Comment by allantito** [comment]
@jonathans199

**Comment by jonathans199** [comment]
@zarela when you have a chance can you please update your repo locally and maybe overwrite the `package-lock.json` the are 43 conflicts, which will take real long to fix. 

So as long as it runs on your computer you could push up the working version, I don't have access to push directly. 

Thank you, let me know any questions.

**Comment by zarela** [comment]
@jonathans199 
I can't override the lock file because we need your new version which is introducing the new package, even if I override you will have a conflict again. What we could do is you delete all your changes for the package-lock file so there are not conflicts there and I can generate a new lock file once your PR is merged

**Comment by jonathans199** [comment]
Closing this merge to main directly, creating a new branch to merge in to staging.

**Pull request closed without merging**